packages/amplify-graphql-auth-transformer/API.md

```diff
@@ -110,6 +110,8 @@ export class AuthTransformer extends TransformerAuthBase implements TransformerA
     // (undocumented)
     addAutoGeneratedRelationalFields: (ctx: TransformerContextProvider, def: ObjectTypeDefinitionNode, allowedFields: Set<string>, fields: readonly string[]) => void;
     // (undocumented)
+    addCustomOperationFieldsToAuthNonModelConfig: (ctx: TransformerTransformSchemaStepContextProvider) => void;
```

Comment:
note: this should be private. Will make it so in a future PR, which will remove it from the published API. Since the transformer interface isn't actually used by customers, this won't be a breaking change